### ISSUE TITLE
Unable to build a .NET Core project targeting Microsoft.NETCore.App

### ISSUE THREAD

**Issue by peterwald** [dotnet/arcade#7562]
A project is included in a mixed solution that targets Microsoft.NETCore.App. When built using arcade, the .NETCore app's build generates an Assembly.runtimeconfig.json file like the below:

```json
{
  "runtimeOptions": {
    "tfm": "netcoreapp3.1",
    "rollForward": "major",
    "framework": {
      "name": "Microsoft.WindowsDesktop.App",
      "version": "3.1.0"
    }
  }
}
```

There is nothing in the project that references windows desktop components and it is intended to be built as a portable .NET app.

My current workaround is to overwrite the Assembly.runtimeconfig.json file with a corrected version after the build completes.

**Comment by markwilkie** [comment]
Hi @peterwald - did you get this sorted?  I don't think this is quite the right team to help, although we'd be happy to try and find someone.

Closing for now, but as I said, we're happy to help.

**Comment by peterwald** [comment]
@markwilkie It looks like it's been fixed after updating our repo to the 6.0 dotnet SDK.
